### ISSUE TITLE
Issues with ESM syntax

### ISSUE THREAD

**Issue by axe312ger** [wix-incubator/kampos#21]
Hey there,

thanks for the quick reply in #20.

I also noticed that the ESM implementation here is outdated, its pretty annoying to get it working with Gatsby (and probably other frameworks that now use/support ESM)

Can you please remove the https://github.com/wix-incubator/kampos/blob/master/package.json#L7? This should already help.

Alternatively: https://rollupjs.org/introduction/#publishing-es-modules (I think you do that, but your RollUp is v0 and v3 is the current... maybe upgrade? :))

Thank you so much,
If you got zero time, I can also give it a try, but didn't do much yet with rollup 🙈

**Comment by ydaniv** [comment]
@axe312ger yep, I'm actually on it now (:

**Comment by ydaniv** [comment]
Hey @axe312ger , here's the WIP PR: https://github.com/wix-incubator/kampos/pull/22
I couldn't remove `type: module`, Rollup keeps screaming at me if I do that. For some reason it keeps complaining of a mismatch between ES modules and Node trying to load CJS 🤷 
But I did add a `module` entry [here](package.json), maybe that will help. Can you check this somehow on your side?

**Comment by axe312ger** [comment]
Yeah if you export it as ESM, you have to keep the `type: module`, so thats fine.

I'll have a look!

**Comment by ydaniv** [comment]
I wonder whether it's ok to use the src index file as endpoint for ESM, or whether a single pre-built dist is also needed. 🤔

**Comment by axe312ger** [comment]
AFAIK: At best you compile down to ES6/ES7 and make sure to keep import/export.

**Comment by ydaniv** [comment]
@axe312ger should be fixed with 16418e3813a63e91fbed26434bce9f3109c7ca7a.
I've published `0.8.0`, please try it out and update here.
Thanks!

**Comment by ydaniv** [comment]
@axe312ger did it work for you?
